ConsoleApp1/Func.cs

```diff
@@ -0,0 +1,80 @@
+﻿using ConsoleApp1;
+namespace func
+{
+    class check
+    {
+        public static int checkInput(string a)
+        {
+            if (int.TryParse(a, out int result)) { return result; }
+            else { return 100; };
+        }
+    }
+
+    class Menu
+    {
+        public static string name = new string("Введите название: ");
+        public static string age = new string("Введите возраст: ");
+        public static string size = new string("Введите размер: ");
+    }
+
+    class Array
+    {
+        public static void InsertInArrFish(ref Fish[] arr, Fish a)
+        {
+            Fish[] newArr = new Fish[arr.Length + 1];
+            for (int i = 0; i < arr.Length; i++)
+            {
+                newArr[i] = arr[i];
+            }
+            newArr[arr.Length] = a;
+
+            arr = newArr;
+        }
+
+        public static void InsertInArrInsect(ref Insects[] arr, Insects a)
+        {
+            Insects[] newArr = new Insects[arr.Length + 1];
+            for (int i = 0; i < arr.Length; i++)
+            {
+                newArr[i] = arr[i];
+            }
+            newArr[arr.Length] = a;
+
+            arr = newArr;
+        }
+
+        public static void InsertInArrAnimal(ref Animal[] arr, Animal a)
+        {
+            Animal[] newArr = new Animal[arr.Length + 1];
+            for (int i = 0; i < arr.Length; i++)
+            {
+                newArr[i] = arr[i];
+            }
+            newArr[arr.Length] = a;
+
+            arr = newArr;
+        }
+
+        public static void showAll(ref Fish[] arrfish, ref Insects[] arrInsects, ref Animal[] arrAnimal)
```

Comment:
аналогично комменту выше, только тут исходные массивы не модифицируются, поэтому ref точно избыточно
```suggestion
        public static void showAll(Fish[] arrfish, Insects[] arrInsects, Animal[] arrAnimal)
```